packages/design-system/src/atoms/inputV2SelectSuffix/index.tsx

```diff
@@ -22,50 +21,51 @@ export const InputV2SelectSuffix: React.FC<InputV2SelectSuffixProps> = ({
     onChange?.(newValue)
   }
 
-  const renderReference: PopoverProps['renderReference'] = useCallback(
-    ({ reference, getReferenceProps, onOpenChange, open, value }) => {
-      const referenceProps = getReferenceProps()
-      const inputProps = Platform.select({
-        web: referenceProps,
-        native: {
-          onFocus: () => {
-            onOpenChange?.(true)
+  const renderReference: Required<SelectV2Props>['renderReference'] =
```

Comment:
Just for the context, is there a reason why useCallback is used here instead of useMemo?

Reply:
Thanks for clarifying!